com.tibia.Tibia.metainfo.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.tibia.Tibia</id>
+
+  <name>Tibia</name>
+  <summary>Become a part of Tibia, the MMORPG classic that has been expanded for more than 25 years</summary>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+
+  <recommends>
+    <display_length compare="ge">1020</display_length>
```

Comment:
The game window cannot be resized below 1020x650. Is this the correct way of using this property?

Reply:
Yea seems fine.